### DISCUSSION TITLE
Master Token: None

### DISCUSSION THREAD

**Discussion by Woorkier** [leikoilja/ha-google-home#754] [Q&A]
I was able to run the Python script in order to get the master token, but i always receive "none" as my master token.
```

[*] Getting master token...
[!] Could not get master token.
[*] Master token: None

[*] Getting access token...
[!] Could not get access token.
[*] Access token: None

[*] Done.
```
Tried using my own password and a generated app password, but none seems to work. Any ideas? Thanks!!

**Comment by leikoilja** [comment]
i will move it into issues https://github.com/leikoilja/ha-google-home/issues/755, @Woorkier
